src/Application/Generator/ArgumentGenerator.php

```diff
@@ -60,6 +70,30 @@ public function generateArguments(array $methodAttributes, string $behaviour = '
                 }
 
                 $attribute = $floatAttributes[$attributeName];
+            } elseif ('DateTimeImmutable' === $typeName) {
```

Comment:
Test Case missing for DateTime and DateTimeImmutable

Reply:
done